### DISCUSSION TITLE
Direct upload to s3 is very slow

### DISCUSSION THREAD

**Discussion by ahmadtibi** [livewire/livewire#4084] [1. Help]
I'm trying to leverage livewire's direct upload to s3 but the upload process takes a very long time, even for a small file size such as 1MB.. my upload speed is 40mbps but the request is taking 7 seconds to complete.

**Comment by joshhanley** [comment]
@ahmadtibi I've personally not had any issues with uploading to S3.

But it's hard to help troubleshoot an issue like that.

Can you do any screen captures to show the issue? Can you also make sure it's uploading directly to S3 and not via a tmp folder on your server?
